src/main/java/core/basesyntax/figures/Figure.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.figures;
+
+public class Figure implements Area, UnitInfo {
+    private String color;
+    private String name;
+
+    public Figure(String color, String name) {
+        this.color = color;
+        this.name = name;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    @Override
+  public double getArea() {
+        return 0;
```

Comment:
The `getArea` method returns 0, which might not be meaningful for a base class. Consider making `Figure` an abstract class if it should not be instantiated directly.